packages/ckeditor5-emoji/src/emojipicker.ts

```diff
@@ -42,16 +42,16 @@ export default class EmojiPicker extends Plugin {
 	/**
 	 * Registered emojis. A pair of an emoji name and all its available skin tone variants.
 	 */
-	private _emojis: Map<string, Array<string>>;
+	public emojis: Map<string, Array<string>>;
+
+	public selectedSkinTone: SkinToneId;
```

Comment:
I would keep it private/protected. Create a getter (fn or just a property) that returns this value.